docroot/themes/humsci/humsci_basic/src/scss/components/_postcard.scss

```diff
@@ -58,42 +62,38 @@
         box-shadow: none;
         color: transparent;
         cursor: pointer;
-      }
-    }
 
-    &.decanter-button {
-      @include unbuttonize;
+        &::before {
+          transform: translateX(0.625rem);
 
-      &:hover {
-        background-color: transparent;
+          @include hb-colorful {
+            @include hb-icon-link-arrow('secondary-darken-12');
+          }
+        }
       }
-    }
 
-    &::before { // horizontal postcard arrow
-      content: '';
-      position: absolute;
-      left: 0;
-      height: hb-calculate-rems(44px); // WCAG AAA target size
-      width: hb-calculate-rems(44px);
+      &::before { // postcard arrow
+        content: '';
+        position: absolute;
+        left: 0;
+        height: hb-calculate-rems(44px); // WCAG AAA target size
+        width: hb-calculate-rems(44px);
 
-      @include hb-colorful {
-        @include hb-icon-link-arrow('secondary');
+        @include hb-colorful {
+          @include hb-icon-link-arrow('secondary');
 
-        transform: translateX(0);
-        transition: transform 200ms ease-in-out;
+          transform: translateX(0);
+          transition: transform 200ms ease-in-out;
+        }
       }
     }
 
-    &:hover,
-    &:focus {
-      cursor: pointer;
-
-      &::before {
-        transform: translateX(0.625rem);
+    &.decanter-button {
```

Comment:
Oh interesting! It looks like the `decanter-button` styles were misplaced. Good find!!